File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    // Prompt for Height
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Print All
    for (int i = 1; i <= height; i++)
    {
        // print gap same as one hash
        for (int j = height; j > i; j--)
        {
            printf(" ");
        }
        // print hashes
        for (int k = i; k > 0; k--)
        {
            printf("#");
        }
        // print gap same as two hash
        printf("  ");
        // print hashes
        for (int k = i; k > 0; k--)
        {
            printf("#");
        }
        // next line
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int


def main():
    # Prompt for Height
    while True:
        height = get_int("Height: ")
        if (height > 0 and height < 9):
            break

    # Print
    for i in range(height):
        print(" " * (height - i - 1), end='')
        print("#" * (i + 1), end='')
        print("  ", end='')
        print("#" * (i + 1))


if __name__ == "__main__":
    main()
